tests/unittest/_torch/speculative/hw_agnostic/test_mtp.py

```
@@ -1769,6 +1784,77 @@ class TargetModel:
         assert not should_use_separate_draft_kv_cache(spec_config)
 
 
+@pytest.mark.parametrize("num_nextn_predict_layers", [2, 3])
+def test_mtp_moe_backend_rejected_after_checkpoint_resolves_one_engine(
+    num_nextn_predict_layers,
+):
+    spec_config = MTPDecodingConfig(
+        max_draft_len=1,
+        speculative_model="/tmp/assistant",
```

Ruff (0.16.1)

[error] Probable insecure usage of temporary file or directory: "/tmp/assistant"

(S108)